docs/docs-content/registries-and-packs/registries/oci-registry/add-pack-oci-basic.md

```diff
@@ -0,0 +1,137 @@
+work in Palette, refer to the [OCI Registry](./oci-registry.md) documentation page.
+
+This guide explains how to upload packs to an OCI registry that supports basic authentication. You will learn how to
+authenticate to your Basic OCI registry, push a custom pack, and configure the registry in Palette.
+
+## Prerequisites
+
+- Tenant administrator access.
+
+- Custom pack files available on your local machine. Refer to the [Add an Add-on Pack](../../adding-add-on-packs.md)
+  guide to learn how to create a custom pack.
+
+- A private OCI registry that supports basic authentication. This guide uses [Harbor](https://goharbor.io/) as an
+  example. Learn how to set up a Harbor registry server using the
+  [Harbor Installation and Configuration](https://goharbor.io/docs/2.9.0/install-config/) guide.
+
+- [ORAS](https://oras.land/docs/installation/) v1.0.0 installed and available.
+
+- [Tar](https://www.gnu.org/software/tar/) installed and available.
+
+- If your OCI registry is using a self-signed certificate or a certificate that is not signed by a trusted Certificate
+  Authority (CA), you will need the certificate to add the registry to Palette.
+
+## Upload Pack to a Basic OCI Registry
+
+Palette supports all OCI-compliant registries that use basic authentication. This guide uses a
+[Harbor](https://goharbor.io/) registry as an example. Follow the steps described below to set up your Harbor registry,
+push the pack, and configure the registry in Palette.
+
+1. Access the Harbor registry server domain on your web browser and log in using your credentials. If you have kept the
+   default credentials, the username and password are **admin** and **Harbor12345**, respectively.
+
+2. From the **Projects** section, click **New Project**. A project in Harbor contains all repositories of an
+   application.
+
+3. Give your project a name and keep the default settings for the remaining configuration. Click **OK** to proceed.
+
+4. In your terminal, export the `HARBOR_ADDRESS` variable, which will store your Harbor address. Do not include the
+   "https://" prefix.
+
+   ```bash
+   export HARBOR_ADDRESS=<Your_Harbor_Address>
+   ```
+
+5. Issue the command `oras login` to log in to your Harbor registry. When prompted, enter your username and password. If
+   the login is successful, you will receive a confirmation message.
+
+   ```bash
+   oras login $HARBOR_ADDRESS
+   ```
+
+6. Next, export the variables required for creating the Harbor repository and pushing the pack.
+
+   - `HARBOR_PROJECT` - Specify a name for the Harbor project that will store the repositories and the pack files. For
+     example, `spectro-oci-registry`.
+   - `NAME` - The pack's name, which must match the `name` parameter in the **pack.json** file.
+   - `VERSION` - The pack's version, which must match the `version` parameter in the **pack.json** file.
+
+   ```bash
+   export HARBOR_PROJECT=<Your_Repository_Name>
```

Comment:
Should this be <your-project-name>? I found the _your repository name_ placeholder a bit confusing 😓

Reply:
Ooops. My bad! Fixed :)